### ISSUE TITLE
TypeError: down.cb is not a function

### ISSUE THREAD

**Issue by riteable** [f-hj/slsk-client#6]
```
/.../slsk-client/lib/index.js:395
            down.cb(new Error('Peer error'))
                 ^

TypeError: down.cb is not a function
    at msgs.forEach.msg (/.../slsk-client/lib/index.js:395:18)
    at Array.forEach (<anonymous>)
    at Messages.forEach (/.../slsk-client/lib/messages.js:41:19)
    at Socket.conn.on.data (/.../slsk-client/lib/index.js:270:10)
    at Socket.emit (events.js:180:13)
    at addChunk (_stream_readable.js:274:12)
    at readableAddChunk (_stream_readable.js:261:11)
    at Socket.Readable.push (_stream_readable.js:218:10)
    at TCP.onread (net.js:581:20)
```

**Comment by f-hj** [comment]
Did you tried to make a download before that? Even in a previous connection, the peer can send an error a long time after.

Corrected in 1.0.8

**Comment by riteable** [comment]
> Did you tried to make a download before that?

That's a possibility. I'll keep that in mind.
